GUI/Main/Main.cs

```diff
@@ -765,17 +767,45 @@ public void LaunchGame()
             var binary = split[0];
             var args = string.Join(" ", split.Skip(1));
 
+            gameTime = new Stopwatch();
+
+            TimeLog t = new TimeLog(CurrentInstance.CkanDir());
             try
             {
+                
                 Directory.SetCurrentDirectory(CurrentInstance.GameDir());
-                Process.Start(binary, args);
+
+                Process kspProcess = new Process();
+                kspProcess.StartInfo.FileName = binary;
+                kspProcess.Exited += new EventHandler(KspExit);
+                kspProcess.EnableRaisingEvents = true;
+
+                kspProcess.Start();
```

Comment:
The `args` parameter is no longer being passed to the process, which will break many common user configs.